### PR TITLE
User details cog 61

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
-  [x] User details .jsx file added 
-  [x] Added showFiled and FormField components to show the details card in show mode

### CODE REVIEW

File: app/javascript/UserDetails/UserDetails.test.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    it('renders PageHeader component', () => {
+      expect(wrapper.find('PageHeader').length).toBe(1);
+    });
+  });
+
+  describe('when label and className props are passed', () => {
+    it('renders the label inside the grid wrapper', () => {
+      wrapper = shallow(<UserDetails />);
+      expect(wrapper.find('ShowField').length).toBe(10);
+      expect(
```
Comment (kelfink) [inline app/javascript/UserDetails/UserDetails.test.js]: 10 is based on the number of users in the Mock service, right?  This spec should not be dependent on the service.  We should have our own locally mocked data.

Comment (Sushmareddy-OSI) [reply app/javascript/UserDetails/UserDetails.test.js]: No, its not number of users , it is number of fields on the card 
{Full Name , Last Login, Start Date, End Date, ......... }

**Comment by kelfink** [comment]
retest this please

File: app/controllers/users_list_controller.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class UsersListController < ApplicationController
```
Comment (kelfink) [inline app/controllers/users_list_controller.rb]: Lint fails....  
[lint:test:rubocop  ] not ok 11 - app/controllers/users_list_controller.rb
[lint:test:rubocop  ] # app/controllers/users_list_controller.rb:1:1: C: Style/FrozenStringLiteralComment: Missing magic comment # frozen_string_literal: true.

File: app/controllers/user_details_controller.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class UserDetailsController < ApplicationController
```
Comment (kelfink) [inline app/controllers/user_details_controller.rb]: Lint style will fail here:  Missing magic comment # frozen_string_literal: true.

File: app/assets/stylesheets/application.css [hunk 4]
```diff
@@ -16,3 +16,14 @@
 @import 'bootstrap/dist/css/bootstrap.css';
 @import 'react-wood-duck/dist/styles/application.css';
 @import 'react-bootstrap-table/css/react-bootstrap-table.css';
+
+.col-md-3 {
+  padding-left: 0;
+}
+.btn.btn-default {
```
Comment (kelfink) [inline app/assets/stylesheets/application.css]: stylelint says we need a blank line here

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -16,3 +16,14 @@
...
+  display: block;
+  font-size: 1.5rem;
+  text-align: right;
+  padding: 2rem 3rem;
+  margin-left: 0rem;
```
Comment (kelfink) [inline app/assets/stylesheets/application.css]: style lint requires we not specify a unit (rem) when the value is 0.  --shrug--

**Pull request merged**